tracer/src/Datadog.Trace.SourceGenerators/Datadog.Trace.SourceGenerators.csproj

```diff
@@ -13,12 +13,8 @@
     <PackageReference Include="System.Reflection.Emit" Version="4.3.0" />
     <PackageReference Include="System.Reflection.Emit.Lightweight" Version="4.3.0" />
   </ItemGroup>
-  <ItemGroup>
-    <Compile Include="..\Datadog.Trace\ClrProfiler\InstrumentationCategory.cs" Link="InstrumentationDefinitions\InstrumentationCategory.cs" />
-    <Compile Include="..\Datadog.Trace\Iast\Dataflow\AspectType.cs" Link="AspectsDefinitions\Sources\AspectType.cs" />
-    <Compile Include="..\Datadog.Trace\Iast\Dataflow\AspectFilter.cs" Link="AspectsDefinitions\Sources\AspectFilter.cs" />
-    <Compile Include="..\Datadog.Trace\Iast\VulnerabilityType.cs" Link="AspectsDefinitions\Sources\VulnerabilityType.cs" />
-    <Compile Include="..\Datadog.Trace\Iast\VulnerabilityTypeName.cs" Link="AspectsDefinitions\Sources\VulnerabilityTypeName.cs" />    
+  <ItemGroup>    
+    <Compile Include="..\..\build\_build\CodeGenerators\InstrumentationCategory.cs" Link="InstrumentationDefinitions\InstrumentationCategory.cs" />    
```

Comment:
I'd prefer we had this the other way personally 🤔 Put the file in `\Datadog.Trace\ClrProfiler\InstrumentationCategory.cs` and then link it into build.csproj and here...